--- elficon-0.6.0/elfres/src/Makefile.am	2015-03-21 12:35:21.692638664 +0200
+++ elficon-0.6.0/elfres/src/Makefile.am	2015-03-21 12:53:05.797670563 +0200
@@ -7,8 +7,7 @@
 	@LIBR_CFLAGS@
 
 bin_PROGRAMS = \
-	elfres \
-	elficon
+	elfres
 
 elfres_SOURCES = \
 	elfres-gui.c \
@@ -26,7 +25,8 @@
 	@ln -s ./elfres ./elficon 2> /dev/null || true
 
 # Add the icon, libglade file, and "gears image" for our binary
-all-local: $(bin_PROGRAMS) $(top_srcdir)/po/de.gmo
+# FIXME: this does not work with libtool
+all-local-disabled: $(bin_PROGRAMS) $(top_srcdir)/po/de.gmo
 	@echo Setting icon for 'elfres'...
 	@cp ./elficon ./elficon-tmp
 	@cp ./elfres ./elfres-tmp
--- /dev/null	2007-02-13 18:29:53.000000000 +0200
+++ ./elfres/src/set-resources.sh	2015-03-21 16:14:38.203852549 +0200
@@ -0,0 +1,26 @@
+#!/bin/sh
+# Usage:
+# set-resources.sh BINARY_TO_MODIFY
+#
+
+# exit on any error
+set -e
+
+target=$1
+
+test -e "$target" || { echo "$target does not exist"; exit 1; }
+
+# use from $PATH, unless set with env var
+: ${ELFRES=elfres}
+: ${ELFICON=elficon}
+
+echo "Setting icon for '$target'..."
+$ELFICON -s $target 1d5481ac-052c-11dd-9a75-7751c40e021a
+$ELFICON -a $target "one-canvas.svg" one-canvas.svg
+$ELFRES -a $target ".glade" elfres.glade
+$ELFRES -a $target ".ui" elfres.ui
+$ELFRES -a $target "gears.svg" gears.svg
+$ELFRES -a $target "gears.png" gears.png
+
+echo "Adding translations for '$target'..."
+$ELFRES -a $target "de/LC_MESSAGES/$target.mo" ${top_srcdir}/po/de.gmo
